digitalpublicgoods/open-concept-lab.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Open Concept Lab",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Unknown"
```

Comment:
Missing import copyright information to prove ownership of the digital solution.